app/models/user.rb

```diff
@@ -1,6 +1,41 @@
 class User < ActiveRecord::Base
+  before_create :grow_beard
+  before_save :hash_password
 
   def fullname
     first_name + " " + last_name
   end
+
+  def trim
+    self.status = "Trimmed"
+    self.save
+  end
+
+  def massage_and_trim
+    self.status = "Massaged and Trimmed"
+    self.save
+  end
+
+  def shave
+    self.status = "Shaved"
+    self.save
+  end
+
+  def grow_beard
+    self.status = "Unshaved beard"
```

Comment:
The status columns is supposed to be a separate table with a relationship. Storing arbitrary strings always cause issues, I accept. But, due to the nature of the application I stored in this way and also since the only entry is controlled by the model  I didn't CamelCased it or underscore it.